### PR TITLE
[PHP 8.4] Add new constants and tokens from the tokenizer

### PR DESCRIPTION
**Pull request by driade**
Once the property hooks page is done, links should be added in tokens.xml

### CODE REVIEW

File: appendices/tokens.xml [hunk 1]
```diff
@@ -729,6 +729,27 @@ defined('T_FN') || define('T_FN', 10001);
...
+     <entry>public(set)</entry>
+     <entry>
+      property hooks (available as of PHP 8.4.0)
+     </entry>
+    </row>
+    <row xml:id="constant.t-protected-set">
+     <entry><constant>T_PROTECTED_SET</constant></entry>
+     <entry>protected(set)</entry>
+     <entry>
+      property hooks(available as of PHP 8.4.0)
```
Comment (jimwins) [inline appendices/tokens.xml:743]: Missing space.

```suggestion
      property hooks (available as of PHP 8.4.0)
```

**Review by jimwins — APPROVED** [review]
LGTM with one nit.

**Comment by driade** [comment]
Done!

**Review by Girgias — CHANGES_REQUESTED** [review]
@iluuu1994 this probably needs to be mentioned in UPGRADING

File: appendices/migration84/constants.xml [hunk 2]
```diff
@@ -299,6 +299,21 @@
...
+    <constant>T_PUBLIC_SET</constant>
+   </member>
+   <member>
+    <constant>T_PROTECTED_SET</constant>
+   </member>
+   <member>
+    <constant>T_PRIVATE_SET</constant>
+   </member>
+  </simplelist>
+ </sect2>
```
[the remark this answers is not in the document]
Comment (driade) [reply appendices/migration84/constants.xml:302]: Done. Reordered tokens.xml too.

**Review by Girgias — APPROVED** [review]

**Pull request merged**